server/chat-plugins/datasearch.ts

```diff
@@ -1509,7 +1509,7 @@ function runMovesearch(target: string, cmd: string, canAll: boolean, message: st
 			}
 			if (matched) continue;
 			if (alts.recoil) {
-				if (dex[move].recoil || dex[move].hasCustomRecoil) matched = true;
+				if (dex[move].recoil) matched = true;
```

Comment:
This will cause dex search to also return Struggle/Mindblown/Steelbeam, which I would imagine is the intention, but I could be wrong

Reply:
Don't worry, it's definitely the intention.

Reply:
"its definitely the intention" is why i think it makes sense to have these as one property, because thats what people *intend* when they think about recoil.  but see below about this, id want to come up with something that also makes sense to researchers

Reply:
This is now *just* showing recoil move by their most pedantic definition, not all self damage moves.